src/xx-apt

```diff
@@ -164,6 +164,13 @@ if [ "$TARGETARCH" = "riscv64" ] && [ "$(xx-info vendor)" = "debian" ]; then
   echo "deb [ arch=riscv64 ] http://ftp.ports.debian.org/debian-ports sid main" >>/etc/apt/sources.list.d/riscv64-sid.list
 fi
 
+if [ "$TARGETARCH" = "loong64" ] && [ "$(xx-info vendor)" = "debian" ]; then
+  apt-get update
+  apt-get install -y debian-ports-archive-keyring
+  echo "deb [ arch=loong64 ] http://ftp.ports.debian.org/debian-ports sid main
```

Comment:
I can't seem to get any packages to install in `sid`. This may be temporary error, but looks like there are conflicts between architectures.

Eg. difference of versions of `gcc-13-base` seems to cause issues.